### ISSUE TITLE
Support for .jp2 files

### ISSUE THREAD

**Issue by pbakes** [4lex4/scantailor-advanced#168]
Single page original or processed JP2 files from archive.org will not load.

**Comment by rudolphos** [comment]
Yes, it's not supported. And I think this project is discontinued.
There now is Scantailor-Universal which has scantailor-advanced built in already and supports jp2. But it's only available on ubuntu yet. If you have WSL2 and vcxsrv you can use it on Windows but it's going to be slower.

https://github.com/trufanov-nok/scantailor-universal
https://github.com/trufanov-nok/scantailor-universal/releases ("Experimental support of importing JPEG 2000 (*.jp2) images via libopenjp2")
